packages/hdwallet-keplr/package.json

```diff
@@ -14,9 +14,9 @@
     "prepublishOnly": "yarn clean && yarn build"
   },
   "dependencies": {
+    "@pastaghost/proto-tx-builder": "^0.7.1-alpha.0",
```

Comment:
can we get this inside the `@shapeshiftoss` namespace, understand if this was just for devving

Reply:
That's a mistake that slipped through from dev. I'll update that now.